spec/models/match_result_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe MatchResult, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
will receive a test, which will also cover that one line here ( https://coveralls.io/builds/14762192/source?filename=app%2Fmodels%2Fmatch_result.rb ), but not today anymore...